tests/test_srcset.py

```diff
@@ -177,8 +190,8 @@ def test_given_width_signs_urls():
 
 def test_given_height_srcset_generates_pairs():
     srcset = _default_srcset({'h': 100})
-    expected_number_of_pairs = 31
-    assert(expected_number_of_pairs == len(srcset.split(',')))
+    expected_dpr_urls = 5
+    assert(expected_dpr_urls == len(srcset.split(',')))
```

Comment:
I know you mentioned this offline, but why is the `expected_dpr_urls` 5 again? Is it because you only generate one URL for each `DPR_TARGET_RATIO` when only `h` is passed as a param?

Reply:
Yes, that's exactly it 👍